dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/client/Client.java

```diff
@@ -26,6 +27,9 @@
 public class Client {
 
     private final OidcClientConfig oidcClientConfig;
+    private static JWKSet jwkSet = null;
+    HttpServletRequest request;
```

Comment:
The same client instance will be used from multiple requests, so storing the request and response will cause concurrency problems. A suggested approach is to pass the request and response objects to the validate method and modify the caller to pass these objects.

Thanks!

Reply:
done